src/main/resources/jenkins/plugins/extracolumns/Messages.properties

```diff
@@ -44,3 +44,5 @@ BuildParametersColumn.DisplayName=Build Parameters
 LastJobConfigurationModificationColumn.DisplayName=Last Configuration Modification
 SlaveOrLabelColumn.DisplayName=Slave Allocation
 UserNameColumn.DisplayName=User Name
+CronTriggerColumn.DisplayName=Periodic Build Trigger
+CronTriggerColumn.ToolTipFormat=Would last have run at {0}; would next run at {1}.
```

Comment:
There is no good reason to copy this mistake and I'm not sure why it would save effort with localization. Better solution: create a pull request and fix it in the job configuration page.